packages/react-celo/src/constants.tsx

```diff
@@ -163,7 +163,7 @@ export const PROVIDERS: {
       </>
     ),
     icon: METAMASK,
-    canConnect: () => isEthereumFromMetamask(),
+    canConnect: () => true,
```

Comment:
this could also be `isMobile ||  isEthereumFromMetamask`

Reply:
Good point.